cpp/ql/test/query-tests/Likely Bugs/Arithmetic/BadAdditionOverflowCheck/PointlessSelfComparison.expected

```diff
@@ -1,9 +1,4 @@
-| templates.cpp:20:5:20:25 | ... < ... | Self comparison. |
-| templates.cpp:21:5:21:25 | ... < ... | Self comparison. |
-| templates.cpp:21:5:21:25 | ... < ... | Self comparison. |
-| templates.cpp:22:5:22:25 | ... < ... | Self comparison. |
-| templates.cpp:22:5:22:25 | ... < ... | Self comparison. |
-| templates.cpp:22:5:22:25 | ... < ... | Self comparison. |
+| templates.cpp:17:5:17:25 | ... < ... | Self comparison. |
```

Comment:
I was wondering why the line number of this result changed after the second commit.  The reason turns out to be that the `.expected` file in the first commit is incorrect - with the correct data the other results are gone and the result on line 17 does not.

Reply:
Ah. I messed up a rebase.